### ISSUE TITLE
Can i speed up yolov3 with tensorrt when my input is video 

### ISSUE THREAD

**Issue by KingWangJL** [lewes6369/TensorRT-Yolov3#56]
I have completed the official example, but when the input of the model is video, the effect is very poor.i need your help ,thans.

**Comment by lewes6369** [comment]
Can you show me your time cost by video? It should be approximate.

**Comment by KingWangJL** [comment]
My model is applied to the nano, we hope to detect hand in real time. And The framework I used is pytorch.

**Comment by lewes6369** [comment]
Do you have tried the tensorRT version besides pytorch?

**Comment by KingWangJL** [comment]
yes,i‘m trying the version of TensorFlow.

**Comment by lewes6369** [comment]
You can try the tiny-yolov3 and runing it by TensorRT.

**Comment by KingWangJL** [comment]
yes,i considered the tiny-yolov3,but the high accuracy is required.

**Comment by GordonDongZHAO** [comment]
> Can you show me your time cost by video? It should be approximate.

Thank you for your work.
I tested with GPU P100.
**Time taken for prepareImage is 272.181 ms.**
Time per inference is 13.0693 ms.
Time taken for nms is 0.009384 ms.
Time taken for postProcessImg is 0.02064 ms.

time for prepareImage may be the bottleneck?
